### PR TITLE
Update README to include comprehensive local deploy, test, and build steps

### PR DESCRIPTION
**Pull request by tylergraff**
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,75 @@ To install the latest developer version (unstable) use:
...
+```
+
+Create a nebari-config.yaml that targets a local kind cluster
+```
+nebari init local --project local-dev --domain example.local --ssl-cert-email [your@email]
+```
+
+Add to hosts file specific resolutions for .local domains specified above  
+```
+echo -e "\n172.18.1.100 local-dev.example.local\n172.18.1.100" example.local\n | sudo tee -a /etc/hosts
```
Comment (krassowski) [inline README.md:196]: This is not always as simple (https://github.com/nebari-dev/nebari-docs/issues/403).

**Review by krassowski — COMMENTED** [review]
I wonder if adding this (or making improvements in general) to https://www.nebari.dev/docs/how-tos/nebari-local/ (in https://github.com/nebari-dev/nebari-docs) would be more maintainable.

**Comment by marcelovilla** [comment]
@tylergraff I also think it makes more sense to add this to the docs rather than the README. Do you have the bandwith to do that?

**Comment by tylergraff** [comment]
Yes, I have bandwidth to move this to the docs. I'll cut a PR there and tag you for review.

**Comment by krassowski** [comment]
FYI there might be some overlap with https://github.com/nebari-dev/nebari-docs/pull/449

**Pull request closed without merging**